# Header. Needed for the frame_id
Header header

nav_msgs/Odometry nav1
nav_msgs/Odometry nav2

geometry_msgs/Twist in_cmd1
geometry_msgs/Twist in_cmd2

geometry_msgs/Twist out_cmd1
geometry_msgs/Twist out_cmd2

geometry_msgs/Point out_cmd_idea1
geometry_msgs/Point out_cmd_idea2

geometry_msgs/Point cmd_geo1
geometry_msgs/Point cmd_geo2

geometry_msgs/Point cmd_aviod1
geometry_msgs/Point cmd_aviod2
